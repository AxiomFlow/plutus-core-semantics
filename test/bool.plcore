(module Foo (imported) (exported () ()) (local () ())
  (define result
    (let
      (lam x
        (case x
          (cl Bool.True  () (con Bool.False))
          (cl Bool.False () (con Bool.True))))
        negate
        [negate [negate (con Bool.True)]])))
