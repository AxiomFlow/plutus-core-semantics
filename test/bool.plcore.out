<T> <k> . </k> <env> ( Foo . result ) |-> ( con Bool . True .ValList ) </env> <mod> Foo </mod> </T>
